packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -160,8 +160,7 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
     this.selected = !this.selected;
   }
 
-  itemClickHandler = (event: MouseEvent): void => {
-    event.preventDefault();
+  itemClickHandler = (): void => {
```

Comment:
If possible, can you make this private? Up to you since we'll do a linting pass to fix these across the board.